packages/block-template/templates/custom-element/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "custom-element-block",
+  "version": "0.0.0",
+  "description": "Block Protocol block template for a custom element-based block",
```

Comment:
```suggestion
  "description": "Block Protocol block template for a custom element-based block",
```
```suggestion
  "description": "Block Protocol block template for a custom-element based block",
```

Reply:
I guess we could go for a triple hyphen?

Reply:
Sorry gotcha, I was going off these things 
<img width="537" alt="image" src="https://user-images.githubusercontent.com/25749103/173697993-ff6a1544-0e8d-404a-8763-35fcc8e4436d.png">
Didn't realise they were also slugs.

I'm good with no hyphens, or triple hyphen. But it's not too important

Reply:
Excuse me, but _I am_ the worst at hyphens, as Ciaran's suggestions on my prior PRs this last fortnight clearly demonstrate